package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/platform-browser": "^7.2.15",
     "@angular/platform-browser-dynamic": "^7.2.15",
     "@angular/router": "^7.2.15",
+    "angular-froala-wysiwyg": "^3.0.0-rc.2-1",
```

Comment:
Minor: Instead of the **rc** version, we could now use their latest stable release (3.0.1-1).